spec/game_spec.rb

```diff
@@ -17,80 +17,173 @@
     end
   end
 
-  describe "#input_is_valid?" do
+  describe '#input_is_valid?' do
     it 'should return false when the input is string version of negative integer' do
-      expect(game.send(:input_is_valid?, "-9")).to eql( false )
+      expect(game.send(:input_is_valid?, '-9')).to eql(false)
     end
 
     it 'should return false when the input is string version of integer that is greater than 9' do
-      expect(game.send(:input_is_valid?, "12")).to eql( false )
+      expect(game.send(:input_is_valid?, '12')).to eql(false)
     end
 
-    it 'should return true when the input is string version of integer between 1 and 9 and the input location is empty' do
-      expect(game.send(:input_is_valid?, "4")).to eql(true)
+    it 'should return true when the input is string version of integer between 1-9 and the input location is empty' do
+      expect(game.send(:input_is_valid?, '4')).to eql(true)
     end
 
     it 'should return false when the input location is occupied' do
       game.board.board['3'] = 'o'
-      expect(game.send(:input_is_valid?, "3")).to eql( false )
+      expect(game.send(:input_is_valid?, '3')).to eql(false)
     end
-
   end
 
-  describe "#any_empty?" do
-    it "should return true if at least one cell in the board is empty" do
+  describe '#any_empty?' do
+    it 'should return true if at least one cell in the board is empty' do
       expect(game.send(:any_empty?)).to eql(true)
     end
 
-    it "should return false if there is no empty cell" do
-      game.board.board['1']='x'
-      game.board.board['2']='x'
-      game.board.board['3']='x'
-      game.board.board['4']='x'
-      game.board.board['5']='x'
-      game.board.board['6']='x'
-      game.board.board['7']='x'
-      game.board.board['8']='x'
-      game.board.board['9']='x'
+    it 'should return false if there is no empty cell' do
+      game.board.board['1'] = 'x'
+      game.board.board['2'] = 'x'
+      game.board.board['3'] = 'x'
+      game.board.board['4'] = 'x'
+      game.board.board['5'] = 'x'
+      game.board.board['6'] = 'x'
+      game.board.board['7'] = 'x'
+      game.board.board['8'] = 'x'
+      game.board.board['9'] = 'x'
       expect(game.send(:any_empty?)).to eql(false)
     end
   end
 
-  describe "#move" do
+  describe '#move' do
     it "should update the board after a player's move, we choose location 2" do
+      puts "For this special test, please enter '2' to select the location in the board !..."
       game.move(player1)
-      expect(game.board.board["2"]).to eql(player1.tag)
+      expect(game.board.board['2']).to eql(player1.tag)
     end
   end
```

Comment:
## Error running test

- Due to `gets.chomp` statement on game.rb file, it's displaying this error 
  - [ ] Remove on game.rb, this statement is only allowed on `main.rb`

![image](https://user-images.githubusercontent.com/24902525/92185504-bff02180-ee19-11ea-9685-33cfb7ec9623.png)

Reply:
We moved ```gets.chomp``` from game.rb to main.rb.